python/correctness/suppressed-exception-handling-finally-break.py

```diff
@@ -0,0 +1,74 @@
+try:
+  for i in range(3):
+    # ruleid: suppressed-exception-handling-finally-break
+    try:
+      1 / 0
+    except ZeroDivisionError:
+      raise ZeroDivisionError("Error: you're trying to divide by zero!")
+    finally:
+      print("finally executed")
+      break
+except ZeroDivisionError:
+  print("outer ZeroDivisionError caught")
+
+
+try:
+  for i in range(3):
+    # ruleid: suppressed-exception-handling-finally-break
+    try:
+      1 / 0
+    except ZeroDivisionError:
+      raise ZeroDivisionError("Error: you're trying to divide by zero!")
+    finally:
+      print("finally executed")
+      continue
+except ZeroDivisionError:
+  print("outer ZeroDivisionError caught")
+
+try:
+  for i in range(3):
+    # ruleid: suppressed-exception-handling-finally-break
+    try:
+      1 / 0
+    except ZeroDivisionError:
+      raise ZeroDivisionError("Error: you're trying to divide by zero!")
+    finally:
+      print("finally executed")
+      return None
```

Comment:
I think this will `SyntaxError` as written:
```
SyntaxError: 'return' outside function
```

Not sure if that's a concern here. It might cause a problem if a future rule is added to look for "`return` outside of function" issues, so it might be worth wrapping this test case in a function to avoid that soft of future issue.